src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionPickerView.xaml

```diff
@@ -0,0 +1,48 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.SolutionPickerView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:local="clr-namespace:NuGet.PackageManagement.UI"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+             x:Name="_solPickerView">
+  <Grid>
+    <TextBlock Text="{x:Static nuget:Resources.Text_OpenInNuGetPackageManager}"
+               FontSize="24"
+               Margin="75,10,0,0"
+               FontWeight="Bold"/>
+    <TextBlock Text="{x:Static nuget:Resources.Text_RecentProjectsSolutions}"
+               FontSize="16"
+               Margin="75,75,0,0"
+               FontWeight="Bold"/>
+    <ListBox ItemsSource="{Binding SolutionList}"
+             Margin="65, 110"
+             BorderThickness="0"
+             HorizontalContentAlignment="Stretch">
+      <ListBox.ItemTemplate>
+        <DataTemplate>
+          <Button
+            Command="{Binding OpenSolutionCommand}"
+            CommandParameter="{Binding FullPath}"
+            IsCancel="True"
```

Comment:
I don't think this is correct. Why are we making the action that is successful the cancel action? This will cause the window to return False for the DialogResult which isn't what we'd want.